bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -434,6 +447,9 @@ class PresentationUploader extends Component {
       <div className={styles.fileList}>
         <table className={styles.table}>
           <tbody>
+            <th className={styles.visuallyHidden} colSpan={3}>{intl.formatMessage(intlMessages.filename)}</th>
```

Comment:
![Screenshot from 2019-05-14 15-17-58](https://user-images.githubusercontent.com/6312397/57725599-8a8b5780-765b-11e9-8ba1-c44667e86c01.png)

What do you see in the console on your end?